### ISSUE TITLE
False positive emitted for ternary statement that involves unreachable code

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#2638]
This is a tracking bug. For the original, refer to https://github.com/microsoft/pylance-release/issues/2123.

```python
# pyright: strict
x = 1
y = "" if True else str(x) # Error: x is unbound
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.192, which I just published. It will also be included in the next release of pylance.
